### PR TITLE
Fix/gopy update

### PR DESCRIPTION
**Pull request by lowaa**
Updating to support only go1.13
Added a test for building and installing the sdist - also tests basic functionality of the lib

### CODE REVIEW

**Review by DanielLeone — APPROVED** [review]
Test passes 🌮 test exists 🎉

File: README.md [hunk 1]
```diff
@@ -196,3 +196,13 @@ If we run the above code and then send some SNMP traps to this listener (in my e
 So to summarise, each call to `get_nowait()` will either raise `Queue.Empty` or return a list of `ReceivedTrap` objects.
 
 The SNMPVariable object is meant to feel like [easysnmp](https://github.com/kamakazikamikaze/easysnmp).
+
+## To run test container
+
+    ./test.sh
```
Comment (smills2929) [inline README.md:202]: :partyparrot:

File: e2e_test/test_built_lib.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+            )
+        ]
+    )
+
+    try:
+        session.connect()
+
+        time.sleep(1)
+
+        send_trap = subprocess.Popen('snmptrap -v 2c -c public 127.0.0.1 '' 1.3.6.1.4.1.8072.2.3.0.1 1.3.6.1.4.1.8072.2.3.2.1',
```
Comment (smills2929) [inline e2e_test/test_built_lib.py:29]: Nice

**Review by smills2929 — APPROVED** [review]
LGTM

**Pull request merged**